src/libostree/ostree-repo-pull.c

```diff
@@ -2508,11 +2508,11 @@ process_one_static_delta (OtPullData                 *pull_data,
    * walk everything twice and keep track of state.
    */
   const guint64 delta_required_blocks = (pull_data->total_deltapart_usize / stvfsbuf.f_bsize);
```

Comment:
If we do this there's no point in doing the `statvfs()` above, instead of `stvfsbuf.f_bsize` we can use `pull_data->repo->txn.blocksize` like the other code does.